hail/python/hailtop/aiogoogle/__init__.py

```diff
@@ -1,5 +1,4 @@
 import warnings
-from ..aiocloud.aiogoogle import *  # noqa: F403
```

Comment:
updated! i just used the `__all__` list from `aiocloud/aiogoogle/__init__.py` to decide what to include, if that seems good

Reply:
I'm hesitant about this because now it needs to be kept in sync with aiocloud.aiogoogle. Why did ruff complain about the star import in the first place? Shouldn't it respect the `noqa`?

Reply:
that.....is a very good point. i reverted this file to the version on `main` and it does not, in fact, complain; it must have at some point? but honestly i don't remember 😅